### PR TITLE
test(integration-test): update web extension test

### PR DESCRIPTION
**Pull request by karliatto**
## Description

Updating web extension integration tests to work with current version of Manifest V2 extension.
Also changing it to use typescript.

* 99534878e3f14fa91509a028bd5cc1be8d842977 - new package for nodejs utilities
* f7e38f128c91c68c96edf55eeb417f35f74ee014 - new package connect extension in replacement of `integration-tests` package that was not used anymore. This package only contain e2e tests but it is intended to be extended with more web extension ready to consume package.
* f0b3ef494bfeb93f63fd08e541d5d8dc38df92ce - adding web extension test to CI in GitLab
* 747d758aeac1b661f27bb552c780d81d92884cd0 - use node utils in connect-popup
* 1012bbde6e92dae50020efe214a3dc1e099be242 - use node utils in suite-desktop

### CODE REVIEW

**Comment by mroz22** [comment]
is it possible to run it somehow in CI?

**Comment by mroz22** [comment]
I am wondering if this is not the obstacle to run it in CI but I guess it should be somehow hackable too https://playwright.dev/docs/chrome-extensions#headless-mode

**Comment by karliatto** [comment]
> I am wondering if this is not the obstacle to run it in CI but I guess it should be somehow hackable too https://playwright.dev/docs/chrome-extensions#headless-mode

Yes, I have already seen that section, and headless works in my machine, but for some reason (I still don't know) in the CI it does not get the page from `browserContext.backgroundPages();` it is undefined. I am still trying few things. If you have any idea let me know :)

File: packages/integration-tests/README.md
Comment (mroz22) [inline packages/integration-tests/README.md:1]: please, change it a little (heading...) and move it to packages/suite/e2e/. could still be useful

File: packages/connect-webextension/package.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    "name": "@trezor/connect-webextension",
+    "version": "1.0.0",
+    "private": true,
+    "scripts": {
+        "lint:js": "yarn g:eslint '**/*.{ts,tsx,js}'",
+        "test:e2e": "yarn xvfb-maybe -- playwright test --config=./playwright.config.ts"
+    },
+    "dependencies": {
+        "@playwright/test": "^1.30.0",
+        "@trezor/node-utils": "workspace:*"
```
Comment (mroz22) [inline packages/connect-webextension/package.json]: maybe only devDependencies?

**Review by mroz22 — APPROVED** [review]
just two nitpicks. otherwise shipit

**Pull request merged**